composer.json

```diff
@@ -20,7 +20,9 @@
             "Keboola\\Processor\\Decompress\\": "src/"
         },
         "files": [
-            "src/getDestinationPath.php"
+            "src/getDestinationPath.php",
+            "src/decompressZip.php",
+            "src/decompressGzip.php"
```

Comment:
udelat z toho classy, at tady toto nemusi byt? - ostatne presun na php-component by to taky resil - getDestinationPath nebude potreba a na decompress muzou byt dve tridy

Reply:
taky mě to napadlo udělat interface a pak ho jednotlivejma dekompresorama implementovat

Reply:
no nebo

Reply:
tak tohle zmizlo

Reply:
ale `getDestinationPath` zůstalo tak jako tak, ono to bere subfoldery a zachovává je to